### ISSUE TITLE
Return YouTube Dislike Not working

### ISSUE THREAD

**Issue by Korinin38** [Anarios/return-youtube-dislike#1103]
### Have you tried to find similar issues (open or recently closed)?

- [X] Yes, this issue is not a duplicate

### Browser

Google Chrome

### Browser Version

130.0.6723.117 (64 bit) (cohort: Stable) 

### Extension or Userscript?

Extension

### Extension/Userscript Version

Version 3.0.0.18

### Video link where you see the problem

any

### What happened?

Does not show dislikes, seems like youtube did something yet again.

The same issue was opened by someone else 4 days ago but then the person who opened the issue closed it without explanation 3 minutes later.

### How to reproduce/recreate?

Open the video.

May be important: I use "Enhancer for YouTube™" ver. 2.0.126.1, though disabling or uninstalling it does not solve the issue.

### Will you be available for follow-up questions to help developers diagnose & fix the issue?

Yes

**Comment by alibabima** [comment]
тоже самое

**Comment by jpa102** [comment]
mine shows up just fine

![image](https://github.com/user-attachments/assets/be4042bd-7290-4435-84e4-51b0ff86b8e2)

extension version: `v3.0.0.18`

**Comment by Monsterovich** [comment]
Maybe this site is blocked in the Russia, because it works fine through a proxy: `https://returnyoutubedislikeapi.com`

**Comment by mihail-jdanov** [comment]
@Monsterovich Tried proxying this domain, it works now. Thank you!

**Comment by User-444** [comment]
It isn't working for me either
<img width="947" alt="Screenshot 2024-11-19 at 11 13 49 AM" src="https://github.com/user-attachments/assets/e6c3ac83-f514-462d-9d36-2c60c6a0edea">

**Comment by jpa102** [comment]
anyone tried using the userscript version of Return YouTube Dislike?

![スクリーンショット (2821)](https://github.com/user-attachments/assets/73ea6c54-45fd-4caa-bb7f-13505d152762)

![image](https://github.com/user-attachments/assets/efacb239-f9de-4b34-90f3-73d7da480f1c)

**Comment by MesaBlack** [comment]
![image](https://github.com/user-attachments/assets/0acac1a1-3050-431e-bd3a-e9a1ada5711f)
 not for me either. Latest Google chrome version.

**Comment by jpa102** [comment]
my config would be:

* Windows 8.1 64 bit
* Google chrome v109.0.5414.168
* Return YouTube Dislike v3.0.0.18

enabling the setting `Disable logging to console` shows that it can load and put the dislikes just fine

i don't know if it's just me though, as my location is in the philippines

**Comment by NicoTheCinderace** [comment]
Bumping this, since it turns out that it's not working today.

**Comment by jpa102** [comment]
trying to help here, can anyone take a screenshot / recording of their devtools console while the youtube watch page loads with return youtube dislike?

![image](https://github.com/user-attachments/assets/eae0c9ce-a4f2-4bb2-941c-5a2b16c98b7b)

**Comment by cepeone** [comment]
Had this issue, went to the extensions page at Chrome Web Store and it was marked as "installed", but stated something like "this extension not working"(can't remember). Uninstalled and added again and it worked after a reload on youtube.

**Comment by PrayStation47** [comment]
> Had this issue, went to the extensions page at Chrome Web Store and it was marked as "installed", but stated something like "this extension not working"(can't remember). Uninstalled and added again and it worked after a reload on youtube.

Can confirm that reinstalling fixed the issue.

**Comment by codeabuse** [comment]
Confirm, add-on doesn't work for several days.
Windows 10, Firefox 133.0.3 (64-bit)

**Comment by GorigoLewis** [comment]
same, no dislikes, like as if i dont have any extension at all

**Comment by GorigoLewis** [comment]
> > Had this issue, went to the extensions page at Chrome Web Store and it was marked as "installed", but stated something like "this extension not working"(can't remember). Uninstalled and added again and it worked after a reload on youtube.
> 
> Can confirm that reinstalling fixed the issue.

nope, i tried

**Comment by GorigoLewis** [comment]
i disabled this and seems like it worked 
![image](https://github.com/user-attachments/assets/f1baf7fe-7422-4414-806b-63189f86b50f)

**Comment by MCLP2005** [comment]
I have the same problem. Though it shows `disabled by Owner`.
---
## System Details:
* OS: `Windows 10 Build 19045.5247`
* Chrome Version: `131.0.6778.205 (Official Build) (64-bit) (cohort: Stable)`
* User Agent: `Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36`
* JavaScript Version: `V8 13.1.201.19`
---
## Plugin Settings
![image](https://github.com/user-attachments/assets/ec3417fc-a1f0-4b6c-943e-4552d0fcd35a)
---
## Interface after refreshing
![image](https://github.com/user-attachments/assets/108502d3-4bf3-4c66-bfb8-f5052677412e)
---
## Log from Plugin in clear text:
```txt
[return youtube dislike]: Initializing smartimation mutation observer
ryd.content-script.js:1069 [return youtube dislike]: Video is loaded. Adding buttons...
ryd.content-script.js:1069 [return youtube dislike]: response from api:
ryd.content-script.js:1069 [return youtube dislike]: {"id":"m4enpnhAavo","dateCreated":"2024-12-29T22:02:03.521611Z","likes":0,"rawDislikes":69,"rawLikes":78,"dislikes":89,"rating":1,"viewCount":24252,"deleted":false}
ryd.content-script.js:1069 [return youtube dislike]: SET dislikes 89
ryd.content-script.js:1069 [return youtube dislike]: likes count disabled by creator
ryd.content-script.js:1069 [return youtube dislike]: removing bar
ryd.content-script.js:1069 [return youtube dislike]: SET dislikes 89
ryd.content-script.js:1069 [return youtube dislike]: likes count disabled by creator
ryd.content-script.js:1069 [return youtube dislike]: removing bar
ryd.content-script.js:1069 [return youtube dislike]: SET dislikes 89
ryd.content-script.js:1069 [return youtube dislike]: likes count disabled by creator
ryd.content-script.js:1069 [return youtube dislike]: removing bar
```
---
If this helps or you need more details, mention me.

**Comment by jpa102** [comment]
> I have the same problem. Though it shows `disabled by Owner`.

from the console logs, isn't that expected behavior? like counts that's been hidden by the creator will show that text

but i can see the data though:

```
{
	"id":"m4enpnhAavo",
	"dateCreated":"2024-12-29T22:02:03.521611Z",
	"likes":0,
	"rawDislikes":69,
	"rawLikes":78,
	"dislikes":89,
	"rating":1,
	"viewCount":24252,
	"deleted":false
}
```

**Comment by bladeofmiquellas** [comment]
Make sure this link opens:  https://returnyoutubedislikeapi.com/votes?videoId=QOFEgexls14 if not try to use VPN.

**Comment by GrayFace** [comment]
> Make sure this link opens: https://returnyoutubedislikeapi.com/votes?videoId=QOFEgexls14 if not try to use VPN.

Yes, adding the site to proxy list has fixed the issue. It didn't load without a proxy. Looks like it could be blocked in Russia.

**Comment by BPplays** [comment]
i was having this issue and it seems disabling ublock origin for youtube fixed it, im not sure if it was a setting i changed but it's worth trying.
